packages/react-scripts/template/src/shared/components/languageSwitcher/languageSwitcher.stories.js

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { actions } from '@storybook/addon-actions';
+import withRouter from 'storybook-react-router';
```

Comment:
What do you think about moving this to `.storybook/decorators`?
I think it'll be more convenient - especially it's not included in plop template and we will have to remember library name. What's more, replacing that solution to a different one will be easier.

Reply:
Agree - moved